multimodal/src/autogluon/multimodal/utils/environment.py

```diff
@@ -6,6 +6,7 @@
 from typing import Dict, List, Optional, Tuple, Union
 
 import torch
+from pytorch_lightning.accelerators import find_usable_cuda_devices
```

Comment:
Seems that it has been added very recently, and is part of `lightning_lite`:

```
from lightning_lite.accelerators.cuda import find_usable_cuda_devices
```

Reply:
Do we need to update our version bound of lightning? https://github.com/autogluon/autogluon/blob/48e1ad051654054f82d033a2c2e3ecc28515bee9/multimodal/setup.py#L40

Reply:
I just checked. The API is only available for `lightning>=1.9.0`:

- 1.9.0: https://github.com/Lightning-AI/lightning/blob/1.9.0/src/pytorch_lightning/accelerators/__init__.py (Has `find_usable_cuda_devices`)
- 1.8.6: https://github.com/Lightning-AI/lightning/blob/1.8.6/src/pytorch_lightning/accelerators/__init__.py (Does not have `find_usable_cuda_devices`)

Reply:
If we need to use this function from lightning, we have to update the version lower bound

Reply:
Lower bound is updated to 1.9.0.